src/components/core/Main/Main.js

```diff
@@ -9,10 +9,12 @@ import {
   Headline,
   Checkbox,
   Input,
+  Textarea,
 } from 'components/common';
 
 
 import {options as selectOptions} from './data';
+import Clock from "../../common/Clock/Clock";
```

Comment:
you need to import from `component/common`
remove it

Reply:
@savtym  Sorry, I missed it for some reason

Reply:
![image](https://user-images.githubusercontent.com/21346502/57958868-01e61300-790a-11e9-80d9-6708319fdcb6.png)

Reply:
I added this to common code for everyone component